### ISSUE TITLE
Verbose shows 1st word only

### ISSUE THREAD

**Issue by andrey-kolegov** [CyCoreSystems/agi#6]
Verbose shows only the 1st word as a message without quotes. Need change:
```
func (a *AGI) Verbose(msg string, level int) error {
-       return a.Command("VERBOSE", msg, strconv.Itoa(level)).Err()
+       return a.Command("VERBOSE", strconv.Quote(msg), strconv.Itoa(level)).Err()
}
```

**Comment by Ulexus** [comment]
Ah, yes.  That sounds perfectly reasonable.  Do you wish to submit a PR, or shall I just make the change?

**Comment by andrey-kolegov** [comment]
I not have permissions for publish. Do it yourself.

**Comment by Ulexus** [comment]
Thanks; released v0.5.2 incorporating this change.
